api/users/views.py

```diff
@@ -708,6 +675,7 @@ class UserSettings(JSONAPIBaseView, generics.RetrieveUpdateAPIView, UserMixin):
 
     required_read_scopes = [CoreScopes.USER_SETTINGS_READ]
     required_write_scopes = [CoreScopes.USER_SETTINGS_WRITE]
+    throttle_classes = (SendEmailDeactivationThrottle, )
```

Comment:
This throttle class only throttles if `deactivation_requested` in payload, otherwise the other attributes can be changed without throttling.